src/extension.ts

```diff
@@ -214,7 +214,13 @@ function initAsyncCommand(context: vscode.ExtensionContext, commandId: string, c
             } else {
                 properties.result = 'Failed';
                 errorData = new ErrorData(err);
-                vscode.window.showErrorMessage(errorData.message);
+                if (errorData.message.includes('\n')) {
+                    util.getOutputChannel().appendLine(errorData.message)
```

Comment:
Should we call outputChannel.show()? I lean yes - just because the output window has multiple 'channels' and it could be hard for users to find the 'App Service' one

Reply:
I agree; it'll also hide that nasty UI glitch where the dropdown menu is black when you click it for the first time.